wordpress/wp-content/plugins/cds-base/classes/Modules/Contact/ContactForm.php

```diff
@@ -41,227 +60,198 @@ public function render($atts, $content = null): string
+                        __('Ask a question.', 'cds-snc'),
+                    ); ?>
+                    <?php $this->radioField(
+                        'goal',
+                        'Get technical support.',
+                        __('Get technical support.', 'cds-snc'),
+                    ); ?>
+                    <?php $this->radioField(
+                        'goal',
+                        'Give feedback.',
+                        __('Give feedback.', 'cds-snc'),
+                    ); ?>
+                    <?php $this->radioField(
+                        'goal',
+                        'Schedule a demo to learn more about GC Articles.',
+                        __('Schedule a demo to learn more about GC Articles.', 'cds-snc'),
+                    ); ?>
+                    <?php $this->radioField(
+                        'goal',
+                        'Other',
+                        __('Other', 'cds-snc'),
+                    ); ?>
+                </div>
+                <!-- end goal of your message -->
 
-                     <!-- Notify - sending_integration -->
-                     <div class="focus-group">
-                     <label class="gc-label required" id="sending-integration-label" for="sending-integration">
-                        <?php _e("Do you require a GC Notify email sending integration? if yes, what for?
-", "cds-snc"); ?>
+                <!-- usage -->
+                <div role="group" aria-labelledby="usage_types">
+                    <label class="gc-label" for="usage" id="usage_types">
+                        <?php _e('What are you thinking about using GC Articles for? (optional)', 'cds-snc'); ?>
                     </label>
-                    <textarea
-                            data-testid="textarea"
-                            class="gc-textarea"
-                            id="sending-integration"
-                            required
-                            placeholder=""
-                            name="sending-integration"
-                    ></textarea>
-                    </div>
-
-                    <div class="buttons">
-                        <button class="gc-button gc-button" type="submit" id="submit">
-                            <?php _e('Submit', 'cds-snc'); ?>
-                        </button>
-                    </div>
-                </form>
-                
-                <?php else : ?>
-                <form id="contact-form" method="POST" action="/wp-json/contact/v1/process">
-                    <?php // echo $heading; ?>
-                    <?php wp_nonce_field(
-                        'contact_form_nonce_action',
-                        'contact',
+                    <p><?php _e('We use this information to improve GC Articles.', 'cds-snc');?></p>
+                    
+                    <div class="focus-group" style="margin-bottom: 1.75rem">
+                    <?php $this->checkBox(
+                        'usage[]',
```

Comment:
Is the "[]" bit significant? I guess it lets us know that there could be multiple values?

Reply:
This will get us an array in PHP
https://stackoverflow.com/questions/4516847/make-array-from-checkbox-form

Reply:
Neat! I didn't know that.